### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by virtualwiz1**
Fixed some typos

## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Comment by virtualwiz1** [comment]
made minor typos and grammar fixed

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Sounds boring. Let's try again.
 
 **Oh My Zsh will not make you a 10x developer...but you may feel like one.**
 
-Once installed, your terminal shell will become the talk of the town _or your money back!_ With each keystroke in your command prompt, you'll take advantage of the hundreds of powerful plugins and beautiful themes. Strangers will come up to you in cafés and ask you, _"that is amazing! are you some sort of genius?"_
+Once installed, your terminal shell will become the talk of the town _or your money back!_ With each keystroke in your command prompt, you'll take advantage of the hundreds of powerful plugins and beautiful themes. Strangers will come up to you in cafés and ask you, _"that is amazing! are you some sort of a genius?"_
```
Comment (mcornella) [inline README.md:9]: This is fine as it is

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ To learn more, visit [ohmyz.sh](https://ohmyz.sh), follow [@ohmyzsh](https://twi
 - [Getting Started](#getting-started)
   - [Prerequisites](#prerequisites)
   - [Basic Installation](#basic-installation)
-    - [Manual inspection](#manual-inspection)
+    - [Manual Inspection](#manual-inspection)
```
Comment (mcornella) [inline README.md:27]: No need to change to Title Case

File: README.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ To learn more, visit [ohmyz.sh](https://ohmyz.sh), follow [@ohmyzsh](https://twi
 - [Advanced Topics](#advanced-topics)
   - [Advanced Installation](#advanced-installation)
     - [Custom Directory](#custom-directory)
-    - [Unattended install](#unattended-install)
+    - [Unattended Install](#unattended-install)
```
Comment (mcornella) [inline README.md:38]: Same as above

File: README.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ To learn more, visit [ohmyz.sh](https://ohmyz.sh), follow [@ohmyzsh](https://twi
 
 ### Basic Installation
 
-Oh My Zsh is installed by running one of the following commands in your terminal. You can install this via the command-line with either `curl`, `wget` or another similar tool.
+Oh My Zsh is installed by running one of the following commands in your terminal. You can install this via the command-line with either `curl`, `wget`, `fetch` or another similar kind of tool.
```
Comment (mcornella) [inline README.md:67]: Not needed to list all available methods and `similar tool` is fine grammar

**Pull request closed without merging**